pkg/services/featuremgmt/registry.go

```diff
@@ -1323,5 +1323,13 @@ var (
 			AllowSelfServe: false,
 			Created:        time.Date(2024, time.January, 10, 12, 0, 0, 0, time.UTC),
 		},
+		{
+			Name:         "nestedFolderPickerOverride",
```

Comment:
naming suggestions welcome - i have no ideas 😂

Reply:
what about just `newFolderPicker`?

Reply:
either way, it doesn't really matter that much I think if we don't intend for anyone to turn this on.